nextflow.config

```diff
@@ -273,23 +273,93 @@ dag {
 
 manifest {
     name            = 'nf-core/pangenome'
-    author          = """Simon Heumos, Michael L Heuer""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
     contributors    = [
-        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
+        [
+            name: 'Philipp Ehmele',
+            affiliation: """Institute of Computational Biology, Helmholtz Zentrum München, Munich, Germany""",
+            email: '',
+            github: '@imipenem',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Gisela Gabernet',
+            affiliation: """Quantitative Biology Center (QBiC) Tübingen, University of Tübingen, Germany""",
+            email: '',
+            github: '@ggabernet',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Erik Garrison',
+            affiliation: """The University of Tennessee Health Science Center, Memphis, Tennessee, TN, USA""",
+            email: '',
+            github: '@ekg',
+            contribution: ['contributor'],
+            orcid: ''
+        ],
+        [
+            name: 'Andrea Guarracino',
+            affiliation: """Genomics Research Centre, Human Technopole, Milan, Italy""",
```

Comment:
My affiliation is the same as Erik

University of Tennessee Health Science Center, Memphis, Tennessee, TN, USA